django/thunderstore/markdown/templatetags/markdownify.py

```diff
@@ -15,6 +15,8 @@
 
 
 def render_markdown(value: str):
+    if value.startswith("\ufeff"):
```

Comment:
Also this could use a test

Reply:
I just added a test case to the existing test (seems codecov doesn't pick that up for some reason), if you'd like me to I can add a full test for it https://github.com/thunderstore-io/Thunderstore/pull/654/files#diff-fa4eb5286b3a081f1fcf4b901eea3c126d0a9ca56b65cfbd0d472fbcdb1e97a2R26

Reply:
Codecov should pick it up unless it never visits that branch, so it could be there's something funky going on with the test itself.

Reply:
You can generate the coverage report locally too fwiw if you do `--cov --cov-report html` for example when running pytest